media/css/firefox/android.less

```diff
@@ -645,13 +645,24 @@ html[dir="rtl"] #phone-wrapper figure {
             border-radius: 0 0 6px 6px;
             border-top: none;
 
-            ul {
+            ul.android {
+                text-align: center;
+
                 li {
-                    float: none;
-                    width: 100%;
+
+                    &.android-all,
+                    &.google-play {
+                        float: none;
```

Comment:
Small nit (and existed before this PR) - looks like `float: none;` is already specified in [send-to-device.less](https://github.com/mozilla/bedrock/blob/master/media/css/base/send-to-device.less#L84).